src/data/customer/__test__/driver.test.js

```diff
@@ -181,65 +195,70 @@ describe('Create new driver ', async () => {
     //   driverTypeIds: [1],
     // };
 
-    const driverInfo = {
-      transactionGroupIds: [180],
-      sendConfirmationSms: false,
-      sendConfirmationEmail: false,
-      driverTypeIds: [2, 3],
-      firstName: 'User',
-      lastName: generateDisplayName(10),
-      email: `${generateDisplayName(10)}@example.com`,
-      password: '123456',
-      birthday: '1980-01-01',
-      phone: `+6592${Math.floor(Math.random() * 1000000)}`,
-      vehicleColor: 'Red',
-      averageSpeed: 60,
-      maximumCapacity: 100,
-      vehicleModelYear: 2018,
-      vehicleLicenseNumber: '12456',
-      vehicleBrand: 'Scooter',
-      vehicleModel: '12456',
-      vehicleTypeId: 1,
-    };
+    it('Test for creating the driver schedule', async () => {
+      const payload = {
+        driverId: driver.id,
+        transactionGroupId: 180,
+        windows: [
+          {
+            startTime: '10:01',
+            endTime: '13:02',
+          },
+        ],
+        startAt: '2020-03-05',
+      };
 
-    try {
-      const responseCreatedriver = await createDriverAsync(
-        driverInfo,
-        token.accessToken
-      );
-      expect('id' in responseCreatedriver).toBeTruthy();
+      schedule = await createDriverScheduleAsync(payload, token.accessToken);
+
+      expect('data' in schedule).toBeTruthy();
+    });
 
+    it('Test for updating the driver schedule', async () => {
       const payload = {
-        driverId: responseCreatedriver.id,
+        driverId: driver.id,
```

Comment:
where does this driver.id come from? before I used the response object from the 'create' @vinay3206

Reply:
@maartenhemmes  it is set from the first test case on this file for creating driver.